js-core/homeworks/phoneApp/src/main.js

```diff
@@ -0,0 +1,111 @@
+function PhoneApp() {
+    this.storage = [];
+}
+
+PhoneApp.prototype.checkNumber = function(numberToCheck) {
```

Comment:
argument called numberToCheck, maybe string to check ? or maybe something like phoneNumber ?